boards/banglejs-2/include/board.h

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright (C) 2019 Inria
+ *               2022 Nicole Faerber
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_banglejs2
+ * @{
+ *
+ * @file
+ * @brief       Board specific configuration for the Bangle.js 2
+ *
+ * @author      Nicole Faebrer <nicole.faerber@digitaluhr-manufaktur.de>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+#include "board_common.h"
+#include "mtd.h"
+#include "periph/pm.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name GPIO pin defines
+ * @{
+ */
+#define LCD_CS                      GPIO_PIN(0, 5)
+// #define LCD_MISO                    GPIO_PIN(0, )
+#define LCD_MOSI                    GPIO_PIN(0, 27)
+#define LCD_SCK                     GPIO_PIN(0, 26)
+#define LCD_EXTCOMIN                GPIO_PIN(0, 6)
+#define LCD_DISP                    GPIO_PIN(0, 7)
+
+#define LCD_BACKLIGHT               GPIO_PIN(0, 8)
+
+#define BUTTON0                     GPIO_PIN(0, 17)
+#define VIBRATOR                    GPIO_PIN(0, 19)
```

Comment:
This is a matter of personal style I guess, but IMO a `_PIN` postfix would make it quite obvious what the define is about. Especially since the the vibratior will also have a PWM dev and channel, such a postfix would make distinguishing easy.